lib/gui/app/pages/finish/templates/success.tpl.html

```diff
@@ -21,24 +21,6 @@
             </svg-icon>
           </span>
         </div>
-        <div class="caption caption-small fallback-footer">
-          made with
-          <svg-icon paths="[ '../../assets/love.svg' ]"
-            width="'auto'"
-            height="'20px'"></svg-icon>
-          by
-          <span os-open-external="https://resin.io?ref=etcher_success">
-            <svg-icon paths="[ '../../assets/balena.svg' ]"
-              width="'auto'"
-              height="'20px'">
-            </svg-icon>
-          </span>
-          <div class="section-footer">
-            <span class="caption caption-small footer-right"
-              manifest-bind="version"
```

Comment:
Does this change mean that the version number won't be displayed anywhere in the application?

Reply:
No, it's gonna be included in the settings, still working on it really, should've made this a `draft` PR